addon-test-support/ember-qunit/index.js

```diff
@@ -34,6 +34,11 @@ export function setupTest(hooks, _options) {
       this.pauseTest = function QUnit_pauseTest() {
         assert.timeout(-1); // prevent the test from timing out
 
+        // This is a temporary work around for
+        // https://github.com/emberjs/ember-qunit/issues/496 this clears the
+        // timeout that would fail the test when it hits the global testTimeout
+        // value.
+        clearTimeout(QUnit.config.timeout);
```

Comment:
This makes sense, as you intentionally _want_ to pause tests indefinitely.

Reply:
Right, I also did confirm that calling `resumeTest()` does continue as expected. So basically clearing the timeout here just means that QUnit itself will wait for the promise to resolve forever....